src/styles/about.scss

```diff
@@ -0,0 +1,18 @@
+.about {
+  display: block;
+  width: 100%;
+
+  &__content {
+    width: 100%;
+    height: 100%;
+
+    object-fit: cover;
+    object-position: center;
+
+    &:hover {
+      transform: scale(1.05);
+  
+      transition: transform 0.3s;
```

Comment:
use it in all cases
![image](https://user-images.githubusercontent.com/91411692/223656142-540c7668-f19b-456a-b66b-76fcd968ab4e.png)
```suggestion
    object-position: center;
  
    transition: transform 0.3s;
    
    &:hover {
      transform: scale(1.05);
```